### ISSUE TITLE
[Accessibility] Provide aria-label to an invisible label where a visible label cannot be used (RM #4772)

### ISSUE THREAD

**Issue by dpage** [postgres/pgadmin4#3019]
***Issue migrated from Redmine: https://redmine.postgresql.org/issues/4772***
*Originally created by **Murtuza Zabuawala** at 2019-09-23 13:58:17 UTC.*

Ref: https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA14.html

**Comment by dpage** [comment]
Issue closed on Redmine.
